Sources/SwiftSyntax/generated/syntaxNodes/SyntaxNodesEF.swift

```diff
@@ -3162,6 +3162,9 @@ public struct FunctionCallExprSyntax: ExprSyntaxProtocol, SyntaxHashable, _LeafE
     _ unexpectedAfterAdditionalTrailingClosures: UnexpectedNodesSyntax? = nil,
     trailingTrivia: Trivia? = nil
   ) {
+    let shouldOmitParens = arguments.isEmpty && trailingClosure != nil
+    let leftParen = leftParen ?? (shouldOmitParens ? nil : .leftParenToken())
+    let rightParen = rightParen ?? (shouldOmitParens ? nil : .rightParenToken())
```

Comment:
Since this code is generated, you won’t be able to modify it. Also, these memberwise initializers should not contain smart logic (eg. consider that maybe you want to omit the parentheses to build an invalid syntax tree for some test case). So, I think any logic that tries to be smart should live only in the convenience initializers.

Reply:
Ah, I see. Thank you for your feedback 😄 
It seems that I can't make a simple modification while maintaining the current structure.
How about declaring a `Bool` property (for example, `shouldCreateBuilderInitializer`) in `CodeGeneration`'s `Node` to indicate whether the result builder-based initializer should be automatically generated, so that it wouldn't be generated for `functionCallExpr`, and instead, manually creating the result builder-based initializer for `functionCallExpr` in [ConvenienceInitializers.swift](https://github.com/swiftlang/swift-syntax/blob/main/Sources/SwiftSyntaxBuilder/ConvenienceInitializers.swift#L194)?
Would it be acceptable to introduce a conditional value just for a specific node?

Reply:
As I said above, I think we should have a way of creating a function call with arguments but without parentheses (eg. to test a syntax tree for invalid Swift code) and this memberwise initializer is the correct one to do. If anything, you could argue that we shouldn’t default `leftParen` and `rightParen` to be `nil`, but I’m not sure that’s all that helpful.

Reply:
So you're saying that the behavior where the result builder-based initializer doesn't insert parentheses tokens by default (in #2912) is intentional.
I understand. I will go ahead and close the PR.
Thank you for the kind explanation 🙇

Reply:
I’m saying that it’s intentional that the non-result builder initializer should have a way to have a `nil` parenthesis but arguments. I’m not as fixed on the result-builder initializers since they already have some special logic, like adding  commas between elements in the result builder.

Reply:
I implemented it by removing the automatically generated code in order to maintain the existing result builder initializer interface. However, if you're reluctant to add an option to `CodeGeneration`'s `Node`, another approach could be to add a convenience initializer that defines the argument label slightly differently 🤔 
It seems that CatchClauseSyntax(https://github.com/swiftlang/swift-syntax/blob/main/Sources/SwiftSyntaxBuilder/ConvenienceInitializers.swift#L93-L107) is already implemented this way.